package.json

```diff
@@ -57,13 +57,15 @@
     "babel-eslint": "^10.1.0",
     "@sendgrid/mail": "^7.4.0",
     "bcrypt": "^5.0.0",
+    "celebrate": "^13.0.4",
     "chai-http": "^4.3.0",
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "express-validator": "^6.7.0",
     "jsonwebtoken": "^8.5.1",
     "mailgen": "^2.0.14",
+    "joi-phone-number": "5.0.0",
```

Comment:
What is the use of this package? I think this one is not needed

Reply:
Did you really read in the codes?